manifests/0000_50_cluster-monitoring-operator_04-config.yaml

```diff
@@ -65,6 +65,12 @@ data:
     # consumers: (@openshift/openshift-team-cluster-manager)
     - '{__name__="cluster_version_available_updates"}'
     #
+    # cluster:usage:resources:sum reports data for cluster usage per resource
+    # Is only used to distinguish ARO for now
+    #
+    # consumers: (@openshift/openshift-team-cluster-manager)
+    - '{__name__="cluster:usage:resources:sum", resource="clusters.aro.openshift.io"}'
```

Comment:
`cluster:usage:resources:sum` is already covered at L30 by `{__name__=~"cluster:usage:.*"}`

Reply:
removed